README.md

```diff
@@ -1,18 +1,75 @@
-# QleetOS User Documentation
+# qleetctl
 
-User docs for QleetOS.
+Install and manage instances of the QleetOS.
 
-## Local Development
+## Requirements
+* [go 1.18](https://go.dev/doc/install)
+* [curl](https://help.ubidots.com/en/articles/2165289-learn-how-to-install-run-curl-on-windows-macosx-linux)
+* [wget](https://www.gnu.org/software/wget/)
+* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [homebrew](https://brew.sh/) MacOS users
 
-Prerequisistes:
+## Install
 
-* [mkdocs](https://www.mkdocs.org/)
```

Comment:
The mkdocs tool is necessary to run the local server when writing docs.  Why remove it from requirements?

Reply:
not sure why it was there to begin with :)